x-pack/plugins/observability_solution/infra/public/common/asset_details_config/asset_details_tabs.tsx

```diff
@@ -90,13 +90,6 @@ const dashboardsTab: Tab = {
   ),
 };
 
-const linkToApmTab: Tab = {
-  id: ContentTabIds.LINK_TO_APM,
```

Comment:
Can you also remove the `ContentTabIds.LINK_TO_APM` and everywhere where it's used? I don't think it's needed anymore.